packages/nodes-base/nodes/Google/Calendar/GoogleCalendarTrigger.node.ts

```diff
@@ -31,6 +31,15 @@ export class GoogleCalendarTrigger implements INodeType {
 		inputs: [],
 		outputs: ['main'],
 		credentials: [
+			{
+				name: 'googleOAuth2Api',
+				required: true,
+				displayOptions: {
+					show: {
+						authentication: ['oAuth2'],
+					},
+				},
+			},
```

Comment:
do we need this for the gmail trigger also?

Reply:
Yeah it's in GmailTrigger too, the file shows only a one-line change since the other stuff (required, displayOptions, etc.) was there for 'GmailOAuth2' which I just changed to GoogleOAuth2Api